packages/core-components/api-report.md

```diff
@@ -700,6 +701,13 @@ export function MissingAnnotationEmptyState(props: Props_3): JSX.Element;
 // @public (undocumented)
 export type MissingAnnotationEmptyStateClassKey = 'code';
 
+// Warning: (ae-missing-release-tag) "MobileSidebar" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
```

Comment:
For anything new that we add, it would be great if you made sure that there are no warnings and no `(undocumented)` for those new things at least.

Also, consider unpacking props/args only inside the actual body of the component/function etc, so that we don't see those internals here in the API and its docs.

```ts
// do
const MyComponent = (props: MyComponentProps) => {
  cons { a, b, c, d, e, f } = props;
// avoid
const MyComponent = ({ a, b, c, d, e, f }: MyComponentProps) => {
```

Reply:
same for the other ones below too of course

Reply:
Good one - tried not to be too repetitive while documenting